jbmc/regression/jbmc/deterministic_assignments_json/clinit-state.json

```diff
@@ -313,10 +313,6 @@
       "@type":"java.lang.Long",
       "value":"9223372036854775807"
     },
-    "inheritedStatic":{
```

Comment:
:question: Is this how it works in the actual script too?

Reply:
It will be. :slightly_smiling_face: 
The "actual script" comments might be confusing for some people. I'll think about how to resolve this and will try to add some more info in a follow-up PR.